tests/tests_manifests_workflow/test_input_manifests_opensearch.py

```diff
@@ -38,26 +38,26 @@ def test_update(self, mock_component_opensearch_min: MagicMock, mock_manifest_to
                     mock_add_to_cron: MagicMock, mock_add_to_versionincrement_workflow: MagicMock,
                     *mocks: MagicMock) -> None:
         mock_component_opensearch_min.return_value = MagicMock(name="OpenSearch")
-        mock_component_opensearch_min.branches.return_value = ["2.12"]
-        mock_component_opensearch_min.checkout.return_value = MagicMock(version="2.12.1000")
+        mock_component_opensearch_min.branches.return_value = ["2.1000"]
```

Comment:
Did not get all 12->1000 changes here.

Reply:
2.12 manifests are moved to legacy folder now, so the code now treat 2.12 as legacy and fail the test 😄 

Move it to 2.1000.x so it is considered as a possible new version to create manifest.